src/point_one/fusion_engine/messages/device.h

```diff
@@ -0,0 +1,64 @@
+/**************************************************************************/ /**
+ * @brief Device status messages.
+ * @file
+ ******************************************************************************/
+
+#pragma once
+
+#include "point_one/fusion_engine/common/portability.h"
+#include "point_one/fusion_engine/messages/defs.h"
+
+namespace point_one {
+namespace fusion_engine {
+namespace messages {
+
+// Enforce 4-byte alignment and packing of all data structures and values.
+// Floating point values are aligned on platforms that require it. This is done
+// with a combination of setting struct attributes, and manual alignment
+// within the definitions. See the "Message Packing" section of the README.
+#pragma pack(push, 1)
+
+/**
+ * @defgroup device_status System Status Message Definitions
+ * @brief Output messages containing device-specific status information.
+ * @ingroup messages
+ *
+ * See also @ref messages.
+ */
+
+/**
+ * @brief System status: GNSS receiver temperature (@ref
```

Comment:
I think just "System status information" is enough. The receiver temperature is just one detail, no need to list out all of the contents (even though for now there's just one).